README.md

```diff
@@ -1852,10 +1852,10 @@ We grant `GameplayAbilities` on the server which then automatically replicates t
 
 The Sample Project stores a `TArray<TSubclassOf<UGDGameplayAbility>>` on the `Character` class that it reads from and grants when the game starts:
 ```c++
-void AGDCharacterBase::AddCharacterAbilities()
+void AGSCharacterBase::AddCharacterAbilities()
 {
 	// Grant abilities, but only on the server	
-	if (Role != ROLE_Authority || !AbilitySystemComponent.IsValid() || AbilitySystemComponent->CharacterAbilitiesGiven)
+	if (Role != ROLE_Authority || !AbilitySystemComponent.IsValid() || AbilitySystemComponent->bCharacterAbilitiesGiven)
```

Comment:
If we change it here, we should change it in the sample project code too.

Reply:
Unless I'm missing something, it's written that way in the sample project code (checked master and 5.1 branches).

Reply:
https://github.com/tranek/GASDocumentation/blob/master/Source/GASDocumentation/Private/Characters/GDCharacterBase.cpp#L277

Reply:
Ah I missed this source code. I was referring to the GASShooter corresponding code. This also explains what you meant at the start with there is no issue with the prefix (which I didn't fully understand/realize you had real code for GASDocumentation but just thought you meant the corresponding code in GAShooter with the prefix change).

I did a PR: https://github.com/tranek/GASDocumentation/pull/95
Sorry for the inconvenience!